### DISCUSSION TITLE
[(ᗜᴗᗜ)]我的xp很明显吗

### DISCUSSION THREAD

**Discussion by WonderofU-awa** [share121/inter-knot#2732] [其他]
![Collage_20240818_180352](https://github.com/user-attachments/assets/7f45c2b0-557c-433b-861a-7f2f8afc7412)
一点都看不出来吧<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by break-under-star** [comment]
啧啧啧，太刀侠真虾头<img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" >

**Comment by LEARoTY** [comment]
虾头太刀<img src="https://bbs-static.miyoushe.com/static/2024/07/31/33f69e7df0fb9d06592379f11d7ba6ba_1797154502027561525.png" alt="云璃 威慑" width="75" >
